descriptions/Engine.Lime_OR_OpenFL.md

```diff
@@ -0,0 +1,3 @@
+[**Lime**](https://lime.software/) is a flexible, lightweight layer for Haxe cross-platform developers. Lime provides a common foundation on native and dynamic targets for window and application life-cycle management, access to rendering and sound, plus many other useful features for cross-platform development.
+
+[**OpenFl**](https://www.openfl.org/) is a free and open-source software framework and platform for the creation of multi-platform applications and video games. OpenFL applications can be written in Haxe, JavaScript or TypeScript.
```

Comment:
Should specify that OpenFL is necessarily built on lime and we are currently unable to distinguish between them, which is why they're combined in one rule.